lib/cli/run-helpers.js

```diff
@@ -175,7 +176,7 @@ const parallelRun = async (mocha, options, fileCollectParams) => {
   debug(
     'executing %d test file(s) across %d concurrent jobs',
     files.length,
-    options.jobs
+    options.jobs ? options.jobs : workerpool.cpus - 1
```

Comment:
I would just remove the job count. It's printed in the debug logs elsewhere:

https://github.com/mochajs/mocha/blob/e677219a78d6deb4bd95ab6a484dff930df7717d/lib/nodejs/buffered-worker-pool.js#L72

maybe just say `executing %d test file(s) in parallel mode`

Reply:
I fixed your feedback. I removed options.jobs params and added text ```in parallel mode```.
Thanks for your comment.